src/components/common/Input/styles.js

```diff
@@ -1,8 +1,41 @@
 import { StyleSheet } from 'react-native';
 
+const baseLabel = {
+  height: 15,
+  textAlign: 'center',
+  fontWeight: 'bold'
+};
+
+const baseInput = {
+  width: 230,
+  height: 40,
+  borderStyle: 'solid',
+  borderWidth: 1,
+  padding: 5,
+  fontWeight: 'bold',
+};
+
+const tomatoe = 'rgb(233, 40, 40)';
```

Comment:
You could have a constants file to store all the styles' constants like this one.